electron-app/src/components/dashboard/BlockchainBalanceCardList.vue

```diff
@@ -0,0 +1,73 @@
+<template>
+  <v-list-item
+    v-if="amount !== zero"
+    :id="`${name}_box`"
+    :ripple="false"
+    to="/blockchain-accounts"
+    class="blockchain-balance-box__item"
+    @click="doNothing"
+  >
+    <v-list-item-avatar tile class="blockchain-balance-box__icon">
+      <crypto-icon
+        width="24px"
+        :symbol="blockchainBalanceIcons[name]"
+      ></crypto-icon>
+    </v-list-item-avatar>
+    <v-list-item-content>
+      <v-list-item-title class="d-flex justify-space-between">
+        <span>
+          {{ name | capitalize }}
+        </span>
+        <span class="text-end">
+          <amount-display
+            show-currency="symbol"
+            fiat
+            :value="amount"
+          ></amount-display>
+        </span>
+      </v-list-item-title>
+    </v-list-item-content>
+  </v-list-item>
+</template>
+
+<script lang="ts">
+import { default as BigNumber } from 'bignumber.js';
+import { Component, Prop, Vue } from 'vue-property-decorator';
+import CryptoIcon from '@/components/CryptoIcon.vue';
+import AmountDisplay from '@/components/display/AmountDisplay.vue';
+import { Zero } from '@/utils/bignumbers';
+
+@Component({
+  components: { AmountDisplay, CryptoIcon }
+})
+export default class BlockchainBalanceCardList extends Vue {
+  @Prop({ required: true })
+  name!: string;
+  @Prop({ required: true })
+  amount!: BigNumber;
+
+  zero = Zero;
+
+  blockchainBalanceIcons = {
+    bitcoin: 'BTC',
+    ethereum: 'ETH'
+  };
+
+  doNothing() {}
```

Comment:
out of curiocity, could we either use that as an arrow function directly in the template or prevent the action any other way? Could also you explain why this is needed maybe?

Reply:
Good find. It's not needed anymore since all of the "list items" do something now, but before it was needed because if you clicked on it the "focus" state wouldn't go away. I will fix!